seeders/usersSeeder.ts

```diff
@@ -0,0 +1,91 @@
+import { PrismaClient } from "@prisma/client";
+import { faker } from "@faker-js/faker";
+import { random } from "./randomFunction";
+
+const prisma = new PrismaClient();
+
+const seedUsers = async (prisma: PrismaClient): Promise<void> => {
+  const users = [];
+  const documentTypes = ["CC", "CE", "PP"];
+  const userRoles = ["user", "restaurantAdmin", "appAdmin"];
+
+  const restaurants = await prisma.restaurants.findMany();
+
+  for (const restaurant of restaurants) {
+    const numberOfAdmins = random(1, 5);
+
+    for (let i = 0; i < numberOfAdmins; i++) {
+      await prisma.users.create({
+        data: {
+          email: faker.internet.email(),
+          password: faker.internet.password(),
+          name: faker.name.firstName(),
+          last_name: faker.name.lastName(),
+          document_type: documentTypes[random(0, documentTypes.length - 1)],
+          document_number: random(100000, 999999999999).toString(),
+          date_of_birth: faker.date.birthdate({
+            min: 18,
+            max: 80,
+            mode: "age",
+          }),
+          city: faker.address.cityName(),
+          contact_email: random(0, 1) === 1 ? true : false,
+          contact_sms: random(0, 1) === 1 ? true : false,
+          contact_wpp: random(0, 1) === 1 ? true : false,
```

Comment:
Why not consider i.e.
```js
contact_wpp: random(0, 1) === 1
```
If the conditional is true, the value is true, else if, it's false, this is not a blocker, but the comparision works.